### DISCUSSION TITLE
Error while running .\rnw-dependencies.ps1

### DISCUSSION THREAD

**Discussion by xiaawan** [microsoft/react-native-windows#9382] [Q&A] [answered]
I am getting this error while running the command .\rnw-dependencies.ps1

======================================================
Start-Process : Cannot validate argument on parameter 'ArgumentList'. The argument is null or empty. Provide an
argument that is not null or empty, and then try the command again.
At C:\ReactNative\rnw-dependencies.ps1:119 char:78
+ ... rgumentList ("modify --channelId $channelId --productId $productId $a ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Start-Process], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Microsoft.PowerShell.Commands.StartProcessCommand
======================================================

How to fix this issue?

**Comment by YanGuang-Chang** [comment]
Did you fix?

**Reply by xiaawan** [reply]
No, I still get the same error.

**Comment by asklar** [comment, marked as answer]
Tracking with Issue #9383
